recipe/meta.yaml

```diff
@@ -24,14 +24,17 @@ requirements:
     - python
     - pytz
     - billiard >=3.6.3,<4.0
-    - kombu >=4.6.8,<4.7
+    - future >=0.18.0
+    - kombu >=4.6.10,<4.7
+    - vine 1.3.0.*
```

Comment:
@igortg, the `vine` dependency was dropped in https://github.com/conda-forge/celery-feedstock/pull/34/files#diff-e178b687b10a71a3348107ae3154e44cL28. Did that happen by accident or was there any particular reasons for that?

Reply:
Sorry for that. Probably by accident. Thanks for fixing it